.github/workflows/test.yaml

```diff
@@ -104,3 +106,49 @@ jobs:
         run: make docker-build-eraser CACHE_FROM=type=gha,scope=eraser-node CACHE_TO=type=gha,scope=eraser-node,mode=max
       - name: Run e2e test
         run: make e2e-test
+
+  scan_vulnerabilities:
+    name: "[Trivy] Scan for vulnerabilities"
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    permissions:
+      contents: read
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+
+      - name: Get tag and repo
+        run: |
+          echo "TAG=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+          echo "REPO=$(echo $GITHUB_REPOSITORY | awk '{print tolower($0)}')" >> $GITHUB_ENV
+
+      - name: Download trivy
+        run: |
+          pushd $(mktemp -d)
+          wget https://github.com/aquasecurity/trivy/releases/download/v${{ env.TRIVY_VERSION }}/trivy_${{ env.TRIVY_VERSION }}_Linux-64bit.tar.gz
+          tar zxvf trivy_${{ env.TRIVY_VERSION }}_Linux-64bit.tar.gz
+          echo "$(pwd)" >> $GITHUB_PATH
+        env:
+          TRIVY_VERSION: "0.28.0"
+
+      - name: Build eraser-manager
+        run: make docker-build-manager \
+          OUTPUT_TYPE=type=docker \
+          CACHE_FROM=type=gha,scope=eraser-manager \
+          CACHE_TO=type=gha,scope=eraser-manager,mode=max \
+          MANAGER_IMG=${{ env.REGISTRY }}/${REPO}-manager:${TAG}
+
+      - name: Build eraser
+        run: make docker-build-eraser \
+          OUTPUT_TYPE=type=docker \
+          CACHE_FROM=type=gha,scope=eraser-node \
+          CACHE_TO=type=gha,scope=eraser-node,mode=max \
```

Comment:
i don't think we need to override OUTPUT_TYPE, CACHE_FROM and CACHE_TO (applies to manager too)

Reply:
ok removed it